pac4j-cas/src/main/java/org/pac4j/cas/client/rest/AbstractCasRestClient.java

```diff
@@ -70,7 +71,7 @@ public void destroyTicketGrantingTicket(final WebContext context, final HttpTGTP
             final URL deleteURL = new URL(endpointURL, endpointURL.getPath() + "/" + profile.getTicketGrantingTicketId());
             connection = HttpUtils.openDeleteConnection(deleteURL);
             final int responseCode = connection.getResponseCode();
-            if (responseCode != HttpStatus.SC_OK) {
+            if (responseCode != HTTP_STATUS_OK) {
```

Comment:
We were pulling in httpclient transitively before. I created constants to remove that dependency.